modules/F50_base_aggregator.sh

```diff
@@ -133,9 +133,14 @@ output_details() {
     DATA=1
   fi
   if [[ "$S22_PHP_VULNS" -gt 0 ]]; then
-    print_output "[+] Found ""$ORANGE""$S22_PHP_VULNS"" issues""$GREEN"" in ""$ORANGE""$S22_PHP_SCRIPTS""$GREEN"" php files.""$NC"
+    print_output "[+] Found ""$ORANGE""$S22_PHP_VULNS"" issues""$GREEN"" in ""$ORANGE""$S22_PHP_SCRIPTS""$GREEN"" php files."
+    print_output "[+] Thereof, found in php.ini files: ""$MAGENTA""$S22_PHP_INI_LIMIT_EXCEEDED"" exceeding the limit value, ""$RED""$S22_PHP_INI_FAILURE"" failure(s) ""$GREEN""and ""$ORANGE""$S22_PHP_INI_WARNINGS"" warnings.""$NC"
```

Comment:
change it to something like this:
print_output "[+] Found ""$ORANGE""$S22_PHP_INI_ISSUES"" issues""$GREEN"" in ""$ORANGE""$S22_PHP_INI_CONFIGS""$GREEN"" php configuratoin files.